deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1591,6 +1591,16 @@ end}.
     {validators, ["non_zero_positive_integer"]}
 ]}.
 
+%% Unsent Message Limit for classic queue consumer channels upon which
+%% the channel blocked.
+%%
+%% {classic_queue_consumer_unsent_message_limit, 200},
+
+{mapping, "classic_queue_consumer_unsent_message_limit", "rabbit.classic_queue_consumer_unsent_message_limit", [
```

Comment:
Please use the [established prefix for CQ-specific features](https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbit/priv/schema/rabbit.schema#L2553).